src/styles/index.scss

```diff
@@ -29,7 +29,7 @@ html {
 			color: $text-color;
 		}
 		a {
-			color: $text-secondary-color;
+			color: $link-color;
```

Comment:
@Aringoaway If I understand it correctly, now all `<a/>` tags will have a `link-color`, right?